contract/models/res_partner.py

```diff
@@ -18,8 +18,7 @@ def _compute_contract_count(self):
         for partner in self:
             partner.contract_count = Contract.search_count([
                 ('recurring_invoices', '=', True),
-                ('partner_id', '=', partner.id),
-                ('date_start', '<=', today),
+                ('partner_id', 'child_of', partner.id),
```

Comment:
IIRC `child_of` expects an array of IDs, so this should be `partner.ids`. I thought maybe this had changed in v11 (because I thought it threw a warning), but a [search of Odoo code backs this up](https://github.com/odoo/odoo/search?utf8=%E2%9C%93&q=child_of&type=).

Reply:
This is still an ID instead of a list.